tests/test_entities/test_questionary.py

```diff
@@ -0,0 +1,99 @@
+from typing import Any, Dict
+
+from pytest_httpx import HTTPXMock
+
+from huntflow_api_client import HuntflowAPI
+from huntflow_api_client.entities import ApplicantsQuestionary
+from huntflow_api_client.models.request.questionary import QuestionaryRequest
+from huntflow_api_client.models.response.questionary import (
+    QuestionaryResponse,
+    QuestionarySchemaResponse,
+)
+from huntflow_api_client.tokens.proxy import HuntflowTokenProxy
+from tests.api import BASE_URL
+
+ACCOUNT_ID = 1
+APPLICANT_ID = 2
+
+ORG_QUESTIONARY_RESPONSE: Dict[str, Any] = {
+    "citizenship": {
+        "type": "dictionary",
+        "id": 1,
+        "title": "Citizenship",
+        "required": False,
+        "order": 1,
+        "value": None,
+        "show_in_profile": True,
+        "dictionary": "citizenship",
+        "disableAutofill": True,
+        "search_filter": True,
+        "account": ACCOUNT_ID,
+        "search_field": "multi_field_1",
+    },
+}
+QUESTIONARY_RESPONSE: Dict[str, Any] = {
+    "citizenship": None,
+    "english": {"id": 1, "name": "B1 – middle", "foreign": "B1", "meta": None},
```

Comment:
field `english` is not present in questionary schema
Tests are a visual aid for using the library, and it's good when they are consistent